src/components/pages/Navbar/Navbar.js

```diff
@@ -99,7 +101,12 @@ const Navbar = ({ userProfile, getProfile, currentUser }) => {
   const accountMenu = <Menu items={menuItems} onClick={handleMenuClick} />;
 
   const reloadLogo = () => {
-    isAuthenticated ? history.push('/') : document.location.reload();
+    //*******************/
+    history.push('/');
+    document.location.reload();
+    //*******************/
+
```

Comment:
Makes sense than if you remove the authentication, it will link the logo to homepage regardless of your auth status. Great job there!